src/activities.py

```diff
@@ -45,6 +70,7 @@ def completeQuiz(self):
         with contextlib.suppress(TimeoutException):
             startQuiz = self.browser.utils.waitUntilQuizLoads()
             self.browser.utils.click(startQuiz)
+        # this is bugged on Chrome for some reason
```

Comment:
If there's a bug, put the version # which you tested on so it can be referred back to in the future.